rtgui/tonecurve.cc

```diff
@@ -91,22 +91,31 @@ ToneCurve::ToneCurve() : FoldableToolPanel(this, "tonecurve", M("TP_EXPOSURE_LAB
     hrenabled = Gtk::manage(new Gtk::CheckButton(M("TP_HLREC_LABEL")));
     hrenabled->set_active(false);
     hrenabled->set_tooltip_markup(M("TP_HLREC_ENA_TOOLTIP"));
-    pack_start(*hrenabled);
 
     method = Gtk::manage(new MyComboBoxText());
     method->append(M("TP_HLREC_LUMINANCE"));
     method->append(M("TP_HLREC_CIELAB"));
     method->append(M("TP_HLREC_COLOR"));
     method->append(M("TP_HLREC_BLEND"));
+    Gtk::VBox *hrVBox;
+    hrVBox = Gtk::manage ( new Gtk::VBox());
```

Comment:
Please replace this with `Gtk::Box` and `new Gtk::Box(Gtk::ORIENTATION_VERTICAL)`

Reply:
@Thanatomanic Ok, sorry for missing that :(

Reply:
No worries, that's what reviews are for 😄